Symfony/CS/Config/Config.php

```diff
@@ -139,6 +140,15 @@ public function getCustomFixers()
         return $this->customFixers;
     }
 
+    public function addCustomFixerSuite(FixerSuiteInterface $suite)
```

Comment:
I would more likely go in direction to add sth like `addCustomFixers` (like we have `addCustomFixer` (without `s` on the end)) that will accept `array|Traversable`